piezo_web_app/PiezoWebApp/tests/integration_tests/delete_job_test.py

```diff
@@ -40,6 +32,7 @@ def test_success_message_is_returned_when_job_deleted_successfully(self):
         body = {"job_name": "test-spark-job", "namespace": "default"}
         kubernetes_response = {'status': 'Success'}
         self.mock_k8s_adapter.delete_namespaced_custom_object.return_value = kubernetes_response
+        self.mock_k8s_adapter.delete_options.return_value = "Options"
```

Comment:
It would be better to use a dictionary here (doesn't need to have all the key-value pairs of the real thing)

Reply:
d63cc7f